eZ/Publish/SPI/Search/Handler.php

```diff
@@ -106,4 +106,23 @@ public function indexLocation(Location $location);
      * @param mixed $contentId
      */
     public function deleteLocation($locationId, $contentId);
+
+    /**
+     * Indexes several content objects.
+     *
+     * @param \eZ\Publish\SPI\Persistence\Content[] $contentObjects
+     */
+    public function bulkIndexContent(array $contentObjects);
```

Comment:
> We should also define exception mapDocumentByContent will throw on db inconsistency issues, this one can contain the relevant warning message for what went wrong, in addition to nesting the given NotFound exceptions etc.

Ignore this part, I was thinking we could define exceptions that where "normal", however it will differ greatly from one search engine to the next, so as you implied on skype, just let the exceptions from `mapDocumentByContent ` fall true for now.